src/sentry/audit_log/register.py

```diff
@@ -0,0 +1,626 @@
+from sentry.models.auditlogentry import AuditLogEntry
+from sentry.utils.strings import truncatechars
+
+from .manager import AuditLogEvent, AuditLogEventManager
+
+default_manager = AuditLogEventManager()
+
+# Cannot use the following as an event_id:
+# 90, 91, 92, 93, 150, 151, and 152
+# These were used for `AuditLogEvent` instances moved to getsentry.
```

Comment:
We generally try to avoid mentioning getsentry in open source repos. Could this be replaced with steps to register a new audit log event? You could add a note there about reserved event_ids, or verifying the event is available in postgres.

Reply:
@mbauer404  @maheskett I feel like we could encode this rule into code for `sentry` in `default_manager.add()` by checking against a disallow set of event ids (i.e. `DISALLOWED_EVENT_IDS = {90, 91, 92, 93, 150, 151, 152}`).

Reply:
@dashed There's already handling for duplicate event ids in the audit log manager. Duplicates should fail tests on merge (@mbauer404 let's confirm), but we don't run getsentry tests on sentry prs.

Reply:
@dashed There's already handling for duplicate event ids in the audit log manager, but I think there could be more test coverage. These event ids are in use in getsentry so shouldn't be disallowed.